daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/PackageInterfaceSpec.scala

```diff
@@ -152,9 +152,140 @@ class PackageInterfaceSpec
     errorCases = Identifier("another package", "Mod:Contract") ->
       Reference.Package("another package"),
     ("AnotherModule:Contract": Identifier) ->
-      Reference.Module(defaultPackageId, "AnotherModule"),
+      Reference.Module(defaultPkgId, "AnotherModule"),
     ("Mod:unit": Identifier) ->
       Reference.Template("Mod:unit"),
   )
 
+  "interfaceImplementations" should {
```

Comment:
Consider adding a test that interface implementations from dependencies do not get included.